Files/wordguess/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+
+namespace GuessGame {
+    class Program {
+        static void Main (string[] args) {
+            string[] lines = File.ReadAllLines (@"C:\Users\Elvira KGB\Documents\wordguess.txt.txt");
```

Comment:
This file does not exist
Also, please use relative paths moving forward. Everyone has a different user home folder